validator-api/Cargo.toml

```diff
@@ -27,35 +27,30 @@ reqwest = { version = "0.11", features = ["json"] }
 rocket = { version = "0.5.0-rc.1", features = ["json"] }
 serde = "1.0"
 serde_json = "1.0"
-tokio = { version = "1.4", features = [
-    "rt-multi-thread",
-    "macros",
-    "signal",
-    "time",
-] }
+tokio = { version = "1.4", features = ["rt-multi-thread", "macros", "signal", "time"] }
 rocket_cors = { git="https://github.com/lawliet89/rocket_cors", rev="dfd3662c49e2f6fc37df35091cb94d82f7fb5915" }
+url = "2.2"
+
+
 anyhow = "1"
 getset = "0.1.1"
-const_format = "0.2.15"
 
 rocket_sync_db_pools = {version = "0.1.0-rc.1", default-features = false}
 sqlx = { version = "0.5", features = ["runtime-tokio-rustls", "sqlite", "macros", "migrate", "time"]}
-coconut-rs = { git = "https://github.com/nymtech/coconut.git", branch = "0.4.0" }
 
 ## internal
 config = { path = "../common/config" }
-crypto = { path = "../common/crypto" }
-gateway-client = { path = "../common/client-libs/gateway-client" }
-mixnet-contract = { path = "../common/mixnet-contract" }
-nymsphinx = { path = "../common/nymsphinx" }
-topology = { path = "../common/topology" }
-validator-client = { path = "../common/client-libs/validator-client" }
-version-checker = { path = "../common/version-checker" }
+crypto = { path="../common/crypto" }
+gateway-client = { path="../common/client-libs/gateway-client" }
+mixnet-contract = { path="../common/mixnet-contract" }
+nymsphinx = { path="../common/nymsphinx" }
+topology = { path="../common/topology" }
+validator-client = { path="../common/client-libs/validator-client", features = ["nymd-client"] }
+version-checker = { path="../common/version-checker" }
```

Comment:
Those changes seem unnecessary, especially since two lines bellow, there are still whitespaces around the `=`

Reply:
Hm, seems my formatter dislikes whitespaces, but at the very least we do need that additional feature for `validator-client`